### DISCUSSION TITLE
Discord Server!

### DISCUSSION THREAD

**Discussion by Jonneal3** [open-webui/open-webui#1736] [General]
Hey Guys - I looked everywhere for a discord on this. Theres none. So Im starting one to discuss things --  https://discord.gg/K5hkY333kz

-J

**Comment by justinh-rahb** [comment]
You didn't look very hard...

https://discord.gg/5rJgQTnV4s

![Screenshot_20240424-204222.png](https://github.com/open-webui/open-webui/assets/52832301/f78f78c9-9e09-412e-89d0-cf48456f235a)

**Reply by entmike** [reply]
FYI there is an official one here: https://discord.gg/5rJgQTnV4s

It's located at the bottom of this page: https://openwebui.com/

**Reply by justinh-rahb** [reply]
And here...

![Screenshot_20240424-205221.png](https://github.com/open-webui/open-webui/assets/52832301/c7459f97-6bed-4afe-b4b0-9bff030011cd)

**Comment by Jonneal3** [comment]
<img width="1920" alt="Screenshot 2024-04-24 at 8 49 57 PM" src="https://github.com/open-webui/open-webui/assets/92951162/094ed4d9-78e0-4989-88bf-1a6384a2448b">

LOL

**Reply by Jonneal3** [reply]
Dam sorry guys just figured discord would jsut show it!

**Comment by tjbck** [comment]
<img width="1077" alt="image" src="https://github.com/open-webui/open-webui/assets/25473318/eef86a56-154d-49b6-94c9-1a165653968f">

Just added to discord discover page, thanks for letting us know :)
